api/api.csproj

```diff
@@ -22,4 +22,40 @@
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
     <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="2.0.0" />
   </ItemGroup>
+  
+  
+  <Target Name="DebugRunWebpack" BeforeTargets="Build" Condition=" '$(Configuration)' == 'Debug' ">
+	<Message Importance="high" Text="Installing any non-peer missing dependencies.." />
+    <Exec Command="npm install" />
+	
+	<!-- It would seem, that the npm install cmd targets the parent directory(natural path of the routine), and yet, the node webpack execution does not. So we go up one level  -->
+    <Message Importance="high" Text="Performing first-run Webpack build..." />
+    <Exec Command="node ../node_modules/webpack/bin/webpack.js --config ../client-react/webpack.config.js" />
+	
+	<Message Importance="high" Text="Starting up docker instances (if not started).." />
```

Comment:
I'm not so sure about running `docker-compose up` as part of the _build_ itself.  It seems like this should be outside of the build process.  Is this standard in Windows/Visual Studio?

Reply:
Have to agree on this aspect too. Was aiming to be all-inclusive, but that too is unnecessary at build time. Will remove